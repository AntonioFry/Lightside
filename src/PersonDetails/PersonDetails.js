import React from 'react';
import { Link } from 'react-router-dom';

const PersonDetails = (props) => {
  console.log(props);
  return(
    <article className='detail-card'>
      <header className='detail-header'>
        <h3>Name:  {props.name}</h3>
        <Link to='/people' className='back-btn'>â—€</Link>
      </header>
      <p>Birthyear: <span className='bold'>{props.birth_year}</span></p>
      <p>Gender: <span className='bold'>{props.gender}</span></p>
      <p>height: <span className='bold'>{props.height}cm</span></p>
      <p>Eye Color: <span className='bold'>{props.eye_color}</span></p>
      <p>Home World:  <span className='bold'></span></p>
    </article>
  )
}

export default PersonDetails;