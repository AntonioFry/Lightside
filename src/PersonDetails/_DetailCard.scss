.detail-card {
  background-color: #fff;
  border: 2px solid black;
  color: black;
  min-width: 250px;
  padding: 5px;
  width: 25%;
}

.detail-header {
  align-items: center;
  border-bottom: 1px dotted black;
  display: flex;
}

.back-btn {
  color: black;
  display: block;
  height: 100%;
  margin: 0px 5px;
  margin: 10px auto;
  text-decoration: none;
}