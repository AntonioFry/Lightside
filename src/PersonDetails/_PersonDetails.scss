.detail-card {
  border: 2px solid black;
  color: black;
  background-color: #fff;
  padding: 5px;
  width: 25%;
}

.detail-header {
  display: flex;
  align-items: center;
  border-bottom: 1px dotted black;
}

.back-btn {
  display: block;
  margin: 10px auto;
  color: black;
  height: 100%;
  margin: 0px 5px;
  text-decoration: none;
}