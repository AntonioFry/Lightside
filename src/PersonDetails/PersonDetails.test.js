import React from 'react';
import PersonDetails from './PersonDetails/'
import { shallow } from 'enzyme';

describe('PersonDetails', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(
      <PersonDetails 
      name='Obi Wan'
      birth_year='1994'
      gender='male'
      height='10cm'
      eye_color='red'
      />
    )
    expect(wrapper).toMatchSnapshot();
  })
});