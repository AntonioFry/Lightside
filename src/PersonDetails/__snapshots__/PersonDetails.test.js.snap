// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonDetails should match the snapshot 1`] = `ShallowWrapper {}`;
